src/components/cylc/TaskFilterSelect.vue

```diff
@@ -6,30 +6,29 @@
     clearable
     multiple
     :placeholder="`Filter by ${type}`"
-    class="mt-2"
-    v-model="localValue.states"
+    v-model="localValue"
   >
     <template #item="{ item, props }">
-      <v-list-item
-        v-bind="props"
-        :title="undefined"
-      >
-        <Workflowicon
-          v-if="type==='workflow state'"
-          :status="item.raw"
-        />
-        <Task
-          v-if="type==='task state'"
-          :task="{ state: item.raw }"
-        />
-        <span class="ml-2">{{ item.raw }}</span>
+      <v-list-item v-bind="props">
+        <template #prepend>
+          <div class="mr-4">
+            <Workflowicon
+              v-if="type==='workflow state'"
+              :status="item.raw"
+            />
+            <Task
+              v-if="type==='task state'"
+              :task="{ state: item.raw }"
+            />
+          </div>
+        </template>
```

Comment:
Rather than re-implementing the title I think it is more natural to only define the `prepend` slot of the `v-list-item`.